packages/@okta/vuepress-site/docs/concepts/inline-hooks/index.md

```diff
@@ -171,3 +171,4 @@ The [Okta System Log](/docs/reference/api/system-log/) captures events related t
 
 <https://developer.okta.com/docs/reference/api/event-types/?q=inline_hook>
 
+> **Note:** You can see both the errors from the error object in the external service response and the errors when Okta can't apply an inline hook response.
```

Comment:
nit: also, there will be errors related to communication with external service (e.g. network related failures, not 200 OK HTTP response)